hack/.deployUtils

```diff
@@ -322,86 +311,82 @@ deployPrometheusForFederation() {
+type: "kuadrant.io/aws"
+stringData:
+  AWS_ACCESS_KEY_ID: ${MGC_AWS_ACCESS_KEY_ID}
+  AWS_SECRET_ACCESS_KEY: ${MGC_AWS_SECRET_ACCESS_KEY}
+  AWS_REGION: ${MGC_AWS_REGION}
+EOF
 
-# Deploy Prometheus in the hub too
-deployPrometheusForFederation ${KIND_CLUSTER_CONTROL_PLANE}
+    cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${KIND_CLUSTER_PREFIX}controller-config
+  namespace: multi-cluster-gateways
+data:
+  AWS_DNS_PUBLIC_ZONE_ID: ${MGC_AWS_DNS_PUBLIC_ZONE_ID}
+  ZONE_ROOT_DOMAIN: ${MGC_ZONE_ROOT_DOMAIN}
+  LOG_LEVEL: "${LOG_LEVEL}"
+EOF
 
-# Deploy Thanos components in the hub
-deployThanos ${KIND_CLUSTER_CONTROL_PLANE}
+    cat <<EOF | kubectl apply -f -
+apiVersion: kuadrant.io/v1alpha1
+kind: ManagedZone
+metadata:
+  name: ${KIND_CLUSTER_PREFIX}dev-mz
+  namespace: multi-cluster-gateways
+spec:
+  id: ${MGC_AWS_DNS_PUBLIC_ZONE_ID}
+  domainName: ${MGC_ZONE_ROOT_DOMAIN}
+  description: "Dev Managed Zone"
+  dnsProviderSecretRef:
+    name: ${KIND_CLUSTER_PREFIX}aws-credentials
+    namespace: multi-cluster-gateways
+    type: AWS
+EOF
+}
 
-# Add workload clusters if MGC_WORKLOAD_CLUSTERS_COUNT environment variable is set
-if [[ -n "${MGC_WORKLOAD_CLUSTERS_COUNT}" ]]; then
-  for ((i = 1; i <= ${MGC_WORKLOAD_CLUSTERS_COUNT}; i++)); do
-    kindCreateCluster ${KIND_CLUSTER_WORKLOAD}-${i} $((${port80} + ${i})) $((${port443} + ${i})) $((${i} + 1))
-    joinSubmarinerBroker ${KIND_CLUSTER_WORKLOAD}-${i}
-    deployIstio ${KIND_CLUSTER_WORKLOAD}-${i}
-    installGatewayAPI ${KIND_CLUSTER_WORKLOAD}-${i}
-    deployIngressController ${KIND_CLUSTER_WORKLOAD}-${i}
-    deployMetalLB ${KIND_CLUSTER_WORKLOAD}-${i} $((${metalLBSubnetStart} + ${i}))
-    deployOLM ${KIND_CLUSTER_WORKLOAD}-${i}
-    deployDashboard ${KIND_CLUSTER_WORKLOAD}-${i} ${i}
-    argocdAddCluster ${KIND_CLUSTER_CONTROL_PLANE} ${KIND_CLUSTER_WORKLOAD}-${i}
-    deployAgentSecret ${KIND_CLUSTER_WORKLOAD}-${i} "true"
-    deployAgentSecret ${KIND_CLUSTER_WORKLOAD}-${i} "false"
-    deployOCMSpoke ${KIND_CLUSTER_WORKLOAD}-${i}
-    deployPrometheusForFederation ${KIND_CLUSTER_WORKLOAD}-${i}
-  done
-fi
+deployQuickStartControl() {
+    clusterName=${1}
+    kubectl config use-context kind-${clusterName}
+    echo "Initialize quickstart setup on ${clusterName}"
+    wait_for "kustomize --load-restrictor LoadRestrictionsNone build ${QUICK_START_HUB_KUSTOMIZATION} --enable-helm --helm-command helm | kubectl apply -f -" "${QUICK_START_HUB_KUSTOMIZATION} cluster config apply" "1m" "5"
+    echo "Waiting for metallb-system deployments to be ready"
+    kubectl -n metallb-system wait --for=condition=ready pod --selector=app=metallb --timeout=300s
+    echo "Waiting for cert-manager deployments to be ready"
+    kubectl -n cert-manager wait --timeout=300s --for=condition=Available deployments --all
+    echo "Waiting for istio deployments to be ready"
+    kubectl -n cert-manager wait --timeout=300s --for=condition=Available deployments --all
```

Comment:
```suggestion
    kubectl -n istio wait --timeout=300s --for=condition=Available deployments --all
```

Reply:
why isio? Or the better question is why it is a `cert-manager` [here](https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/hack/quickstart-setup.sh#L171-L181) ?

Reply:
yeah, it's an error in the previous pr. It should be istio-operator. I can fix it seperately if you prefer.